pkg/apis/clouddns/v1/zone_types.go

```diff
@@ -4,7 +4,7 @@ import "time"
 
 type Revision struct {
 	CreatedAt  time.Time `json:"created_at"`
-	Identifier string    `json:"identifier"`
+	Identifier string    `json:"identifier" anxcloud:"identifier"`
```

Comment:
The `Revision` type is not an `Object`, so it shouldn't be used by anything requiring this tag

Is there an Engine API to CRUD a specific revision? If so, adding this might still make sense

Reply:
welp, likely I applied the wrong solution to the problem. I think this PR can be closed then. ^^